README.md

```diff
@@ -129,6 +129,22 @@ The `Bootstrap` class and `bootstrap/` template path are deprecated since 2.0
 and will be removed in 3.0.
 
 
+## Overview of icons
```

Comment:
I think it's better to move this section to the README file under the examples folder.

Besides, it will be helpful to mention the icons page in the icon page.

Reply:
done